kythe/web/site/examples.md

```diff
@@ -122,17 +115,18 @@ java -Xbootclasspath/p:third_party/javac/javac*.jar \
 ## Indexing the Kythe Repository
 
 {% highlight bash %}
-mkdir .kythe_{graphstore,compilations}
+mkdir -p .kythe_{graphstore,compilations}
 # .kythe_serving is the output directory for the resulting Kythe serving tables
 # .kythe_graphstore is the output directory for the resulting Kythe GraphStore
-# .kythe_compilations will contain the intermediary .kindex file for each
+# .kythe_compilations will contain the intermediary .kzip file for each
 #   indexed compilation
 
-# Produce the .kindex files for each compilation in the Kythe repo
+# Produce the .kzip files for each compilation in the Kythe repo
 ./kythe/extractors/bazel/extract.sh "$PWD" .kythe_compilations
 
 # Index the compilations, producing a GraphStore containing a Kythe index
-docker run --rm \
+bazel build //kythe/release:docker
+docker run --rm -ti \
```

Comment:
Is the `-ti` flag needed? I've only used that before when running something interactive in the container.